digital-deck-ui/src/store/index.js

```diff
@@ -53,6 +58,7 @@ export default createStore({
                 commit('setPlayerCards', res.data.players[0].cards);
                 commit('setTableCards', res.data.table);
                 commit('setCardsInDeck', res.data.deck);
+                commit('setNumPlayers', res.data.numPlayers);
```

Comment:
Instead of `numPlayers`, I think it would be more practical if we keep an array of player names instead.

Reply:
This again will require a lot of change when handling requests and sending data, which in turn may make our tests fail.

Reply:
We'll have to modify the tests in that case, since they are not useful for the current state of our app.